common.yml

```
@@ -234,129 +234,129 @@ translations:
 
     # features
     - key: options.features.never_heard
-      t: <span aria-hidden="true">🤷</span> Never heard of it/Not sure what it is
+      t: <span aria-hidden="true">🤷</span> Nunca ouvi falar/Não tenho certeza do que é
     - key: options.features.heard
-      t: <span aria-hidden="true">✅</span> Know what it is, but haven't used it
+      t: <span aria-hidden="true">✅</span> Eu sei o que é mas nunca usei
     - key: options.features.used
-      t: <span aria-hidden="true">👍</span> I've used it
+      t: <span aria-hidden="true">👍</span> Já usei
 
     # features (no emoji)
     - key: options.features.never_heard.label
-      t: Never heard of it/Not sure what it is
+      t: Nunca ouvi falar/Não tenho certeza do que é
     - key: options.features.heard.label
-      t: Know what it is, but haven't used it
+      t: Eu sei o que é mas nunca usei
     - key: options.features.used.label
-      t: I've used it
+      t: Já usei
 
     # features (short versions)
     - key: options.features.never_heard.short
-      t: Never heard of it
+      t: Nunca ouvi falar
     - key: options.features.heard.short
-      t: Know about it
+      t: Conheço
     - key: options.features.used.short
-      t: Have used it
+      t: Já usei
 
     # patterns
     - key: options.patterns.use_never
-      t: Almost always avoid
+      t: Quase sempre evitar
     - key: options.patterns.use_sparingly
-      t: Use sparingly
+      t: Usar moderadamente
     - key: options.patterns.use_neutral
-      t: Neutral
+      t: Neutro
     - key: options.patterns.use_frequently
-      t: Use frequently
+      t: Usar frequentemente
     - key: options.patterns.use_always
-      t: Use as much as I can
+      t: Usar o máximo que puder
```

Eu não consegui encontrar no site onde essas _strings_ são usadas, então não pude confirmar. Estou em dúvida se não seria melhor utilizar:

```suggestion
      t: Usei o máximo que pude
```